lib/utils/isStandardSyntaxRule.js

```diff
@@ -1,15 +1,13 @@
+/* @flow */
 "use strict"
 
 const _ = require("lodash")
 const isCustomPropertySet = require("../utils/isCustomPropertySet")
 
 /**
- * Check whether a rule is standard
- *
- * @param {Rule} postcss rule node
- * @return {boolean} If `true`, the rule is standard
+ * Check whether a Node is a standard rule
  */
-module.exports = function (rule) {
+module.exports = function (rule/*: Object*/)/*: boolean*/ {
```

Comment:
@m-allanson The fact that we check for `rule.params` below means it's confusing — might not actually be a rule.